php/php.editor/src/org/netbeans/modules/php/editor/verification/AddOverrideAttributeHint.java

```diff
@@ -160,6 +160,10 @@ private void addAddOverrideHints(CheckVisitor checkVisitor, List<Hint> hints, Ba
                 return;
             }
             Identifier methodName = method.getFunction().getFunctionName();
+            if(methodName.getName().equals("__construct")){
```

Comment:
Please use `CodeUtils.isConstructor(method)`. And put the statement before the above line.
As I wrote before, please format it following the existing source code.(Add whitespaces after `if` and before `{`)